src/main.rs

```diff
@@ -82,10 +84,11 @@ async fn main() {
             conways.toggle_state_cell((mark_cell_x, mark_cell_y));
         }
 
-        if running && get_time() - last_updated > UPDATE_TIMER {
-            last_updated = get_time();
+        if running && accum_delta >= 2. * TICK_RATE {
```

Comment:
Why is this checking if the delta is twice the tick_rate, wouldn't it be easier double the `TICK_RATE` const?